miso-web/src/main/webapp/scripts/list_study.js

```diff
@@ -61,7 +61,12 @@ ListTarget.study = {
         }
       }, ];
     } else {
-      return [];
+      return [{
+          "name": "Add",
+          "handler": function() {
+            window.location = "/miso/study/new";
```

Comment:
Could prevent the duplication here by removing the if/else and just adding to the URL if `projectId` is defined (`'/miso/study/new' + projectId ? ('/' + projectId) : ''`)